sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -490,6 +496,149 @@ public void tearDown() {
+    schemaBuilder.addField(Schema.Field.of("column_int", Schema.FieldType.INT32));
+    schemaBuilder.addField(Schema.Field.of("column_long", Schema.FieldType.INT64));
+    schemaBuilder.addField(Schema.Field.of("column_float", Schema.FieldType.FLOAT));
+    schemaBuilder.addField(Schema.Field.of("column_double", Schema.FieldType.DOUBLE));
+    schemaBuilder.addField(Schema.Field.of("column_bigdecimal", Schema.FieldType.DECIMAL));
+    schemaBuilder.addField(Schema.Field.of("column_date", LogicalTypes.JDBC_DATE_TYPE));
+    schemaBuilder.addField(Schema.Field.of("column_time", LogicalTypes.JDBC_TIME_TYPE));
+    schemaBuilder.addField(Schema.Field.of("column_timestamptz", LogicalTypes.JDBC_TIMESTAMP_WITH_TIMEZONE_TYPE));
+    schemaBuilder.addField(Schema.Field.of("column_timestamp", Schema.FieldType.DATETIME));
+    schemaBuilder.addField(Schema.Field.of("column_short", Schema.FieldType.INT16));
+    Schema schema = schemaBuilder.build();
+
+    String tableName = DatabaseTestHelper.getTestTableName("UT_WRITE");
+    StringBuilder stmt = new StringBuilder("CREATE TABLE ");
+    stmt.append(tableName);
+    stmt.append(" (");
+    stmt.append("column_boolean       BOOLEAN,");             // boolean
+    stmt.append("column_string        VARCHAR(254),");        // String
+    stmt.append("column_int           INTEGER,");             // int
+    stmt.append("column_long          BIGINT,");              // long
+    stmt.append("column_float         REAL,");               // float
+    stmt.append("column_double        DOUBLE PRECISION,");    // double
+    stmt.append("column_bigdecimal    DECIMAL(13,0),");       // BigDecimal
+    stmt.append("column_date          DATE,");                // Date
+    stmt.append("column_time          TIME,");                // Time
+    stmt.append("column_timestamptz   TIMESTAMP,");           // Timestamp
+    stmt.append("column_timestamp     TIMESTAMP,");           // Timestamp
+    stmt.append("column_short         SMALLINT");             // short
+    stmt.append(" )");
+    DatabaseTestHelper.createTableWithStatement(dataSource, stmt.toString());
+    try {
+      ArrayList<Row> data = getRowsToWrite(rowsToAdd, schema);
+      pipeline.apply(Create.of(data)).setRowSchema(schema).apply(JdbcIO.<Row>write()
+              .withDataSourceConfiguration(
+                      JdbcIO.DataSourceConfiguration.create(
+                              "org.apache.derby.jdbc.ClientDriver",
+                              "jdbc:derby://localhost:" + port + "/target/beam"))
+              .withBatchSize(10L)
+              .withTable(tableName)
+      );
+      pipeline.run();
+      assertRowCount(tableName, rowsToAdd);
+    } finally {
+      DatabaseTestHelper.deleteTable(dataSource, tableName);
+    }
+  }
+
+
+  @Test
+  public void testWriteWithoutPreparedStatementAndNonRowType() throws Exception {
+    final int rowsToAdd = 10;
+
+    String tableName = DatabaseTestHelper.getTestTableName("UT_WRITE");
```

Comment:
Isn't this the same table name used by other tests? Might be a good idea to use a different name to avoid any weird bugs during parallel test runs.